Tools/CISupport/ews-build/steps.py

```diff
@@ -158,6 +158,7 @@ class GitHubMixin(object):
     addURLs = False
     pr_open_states = ['open']
     pr_closed_states = ['closed']
+    SKIP_EWS_LABEL = 'skip-ews'
```

Comment:
Don't we need to change our hook processing so that _removing_ this label re-triggers EWS?